src/objective-c/tests/Podfile

```diff
@@ -4,11 +4,11 @@ platform :ios, '8.0'
 install! 'cocoapods', :deterministic_uuids => false
 
 def shared_pods
-	pod 'Protobuf', :path => "../../../third_party/protobuf"
-	pod 'BoringSSL', :podspec => ".."
-	pod 'CronetFramework', :podspec => ".."
-	pod 'gRPC', :path => "../../.."
-	pod 'RemoteTest', :path => "RemoteTestClient"
+  pod 'Protobuf', :path => "../../../third_party/protobuf", :inhibit_warnings => true
+  pod 'BoringSSL', :podspec => "..", :inhibit_warnings => true
```

Comment:
We should open issues in the Protobuf repo for the warnings we got. Same for the BoringSSL guys, although they don't use GitHub.